### PR TITLE
added is_bookable to production

### LINKED ISSUE #68: production is_bookable

### PR DESCRIPTION
**Pull request by streth11**
closes #68

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -191,6 +191,7 @@ type ProductionNode implements Node {
   performances(offset: Int, before: String, after: String, first: Int, last: Int, id: ID, start: DateTime, start_Year_Gt: DateTime): PerformanceNodeConnection!
   start: DateTime
   end: DateTime
+  isBookable: Boolean
```
Comment (streth11) [inline schema.graphql:194]: why is this not "Boolean!"

File: uobtheatre/productions/schema.py [hunk 2]
```diff
@@ -66,12 +66,17 @@ class ProductionNode(GrapheneImageMixin, DjangoObjectType):
     start = graphene.DateTime()
     end = graphene.DateTime()
 
+    is_bookable = graphene.Boolean()
```
Comment (JElgar) [inline uobtheatre/productions/schema.py:69]: I think might make it an exclimation mark
```suggestion
    is_bookable = graphene.NonNull(graphene.Boolean())
```

**Review by JElgar — APPROVED** [review]
If you want an exclamation mark I think that might do the trick... But honestly, I don't think I care

File: uobtheatre/productions/schema.py (hunk 2 above)
Comment (JElgar) [reply uobtheatre/productions/schema.py:69]: Or maybe even
```suggestion
    is_bookable = graphene.Boolean(required=True)
```

File: schema.graphql [hunk 3]
```diff
@@ -191,6 +191,7 @@ type ProductionNode implements Node {
   performances(offset: Int, before: String, after: String, first: Int, last: Int, id: ID, start: DateTime, start_Year_Gt: DateTime): PerformanceNodeConnection!
   start: DateTime
   end: DateTime
+  isBookable: Boolean!
```
Comment (JElgar) [inline schema.graphql:194]: :tada:

**Review by JElgar — COMMENTED** [review]
LGTM

**Pull request merged**